qemu/tests/cfg/cpu_add.cfg

```diff
@@ -115,10 +115,9 @@
             offline = 1
             vcpu_num_rechek = no
             ntp_server = "clock.redhat.com"
-            pre_command = "ntpdate ${ntp_server}"
-            ntp_sync_cmd = "ntpdate ${ntp_server}"
-            ntp_query_cmd = "ntpdate -q ${ntp_server}"
-            ntp_service_stop_cmd = "service chronyd stop || service ntpdate stop"
+            ntp_sync_cmd = "(systemctl stop chronyd || service ntpdate stop)"
+            ntp_sync_cmd += " && chronyd -q 'server clock.redhat.com iburst'"
```

Comment:
Looks like previous step is to sync first and then stop, now it reverses, is this what we expected?

Reply:
Yes, it is the same.